### PR TITLE
add blur estimation example

### PR DESCRIPTION
**Pull request by 4BooM04**

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -382,10 +390,55 @@ class _MainPageWidgetState extends State<MainPageWidget> {
...
+      }
+    } catch (e) {
+      Logger.root.severe(e);
     }
   }
 
+  void showResultTextDialog(result) {
+    Widget okButton = FlatButton(
+      child: Text("OK"),
+      onPressed: () {},
```
Comment (EKnizhnik) [inline lib/main.dart:422]: The dialog will probably be undismissible.

File: pubspec.lock [hunk 2]
```diff
@@ -229,10 +236,10 @@ packages:
   scanbot_sdk:
     dependency: "direct main"
     description:
-      name: scanbot_sdk
-      url: "https://pub.dartlang.org"
-    source: hosted
-    version: "2.2.2"
+      path: "../scanbot-sdk-flutter"
```
Comment (MrDyro) [inline pubspec.lock:239]: The public example shouldn't use a local SDK version

File: pubspec.yaml [hunk 3]
```diff
@@ -26,6 +26,7 @@ dependencies:
   image_picker: ^0.6.7+21
   path_provider: ^1.3.0
   shared_preferences: 0.5.12+4
+  logging: ^0.11.4
```
Comment (MrDyro) [inline pubspec.yaml:29]: Update Libraries.txt

**Review by MrDyro — CHANGES_REQUESTED** [review]

File: pubspec.lock (hunk 2 above)
Comment (EKnizhnik) [reply pubspec.lock:239]: I can suggest that this is for testing purposes. We can't merge this until the release, as far I understand.

File: lib/main.dart [hunk 4]
```diff
@@ -382,10 +396,58 @@ class _MainPageWidgetState extends State<MainPageWidget> {
...
+  }
+
+  estimateBlurriness() async {
+    if (!await checkLicenseStatus(context)) {
+      return;
+    }
+    try {
+      var image = await ImagePicker().getImage(source: ImageSource.gallery);
+
+      ///before processing image sdk need storage read permission
```
Comment (MrDyro) [inline lib/main.dart:410]: Typo: image -> an image / sdk need -> the SDK needs

**Review by MrDyro — CHANGES_REQUESTED** [review]

File: pubspec.yaml (hunk 3 above)
Comment (eddyfrank) [reply pubspec.yaml:29]: Libraries.txt should only contain libs of the SDK, not of the app.

Comment (4BooM04) [reply pubspec.yaml:29]: made a copy of the library file from  flutter sdk

**Pull request merged**